src/lib/core/coercion/date-property.ts

```diff
@@ -0,0 +1,5 @@
+/** Coerces a data-bound value (typically a string) to a Date. */
+export function coerceDateProperty(value: any, fallbackValue = new Date()): Date {
+  let timestamp = Date.parse(value);
```

Comment:
moment is huge, there is ongoing discussion as to how to make it more lightweight https://github.com/moment/moment/issues/2373 there a pretty popular ES6 friendly alternative https://github.com/date-fns/date-fns